### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by Anirudh4Neutrino**
Shooter Pull Request

### CODE REVIEW

File: 2024 Code/src/main/java/frc/robot/commands/ShooterDefaultCommand.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.util.SubsystemContainer;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class ShooterDefaultCommand extends Command {
+    @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
+
+    /**
```
Comment (ViennaRossmanith) [inline 2024 Code/src/main/java/frc/robot/commands/ShooterDefaultCommand.java:11]: get rid of these comments

File: 2024 Code/src/main/java/frc/robot/commands/SwerveDefaultCommand.java [hunk 2]
```diff
@@ -17,6 +17,10 @@ public SwerveDefaultCommand(CommandXboxController p_controller) {
     addRequirements(SubsystemContainer.swerveSubsystem);
   }
 
+  public SwerveDefaultCommand() {
+    // TODO Auto-generated constructor stub
+  }
```
Comment (ViennaRossmanith) [inline 2024 Code/src/main/java/frc/robot/commands/SwerveDefaultCommand.java]: remove these comments

File: 2024 Code/src/main/java/frc/robot/util/SubsystemContainer.java [hunk 3]
```diff
@@ -4,13 +4,15 @@
...
 import frc.robot.subsystems.LimelightSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
 import frc.robot.subsystems.SwerveSubsystem;
 
 public class SubsystemContainer {
     public static final LimelightSubsystem limelightSubsystem = new LimelightSubsystem();
     public static final LEDSubsystem LEDSubsystem = new LEDSubsystem();
     public static final SwerveSubsystem swerveSubsystem = new SwerveSubsystem();
     public static final IntakeSubsystem intakeSubsystem = new IntakeSubsystem();
+    public static final ShooterSubsystem ShooterSubsystem = new ShooterSubsystem();
```
Comment (ViennaRossmanith) [inline 2024 Code/src/main/java/frc/robot/util/SubsystemContainer.java:15]: lowercase s

**Review by ViennaRossmanith — COMMENTED** [review]
remove the unnecessary comments

**Review by CaleWineinger — APPROVED** [review]

**Pull request merged**